client/omnichannel/directory/chats/contextualBar/ChatInfo.js

```diff
@@ -38,15 +38,15 @@ const DepartmentField = ({ departmentId }) => {
 
 const ContactField = ({ contact, room }) => {
 	const t = useTranslation();
-	const { username, status } = contact;
+	const { status } = contact;
 	const { fname, t: type } = room;
 	const avatarUrl = roomTypes.getConfig(type).getAvatarPath(room);
 
 	return <>
 		<Label>{t('Contact')}</Label>
 		<Info style={{ display: 'flex' }}>
 			<Avatar size='x40' title={fname} url={avatarUrl} />
-			<UserCard.Username mis='x10' name={username} status={<UserStatus status={status} />} />
+			<UserCard.Username mis='x10' name={fname} status={<UserStatus status={status} />} />
```

Comment:
We need to display contact's **name** or **username**, not room's data.

Reply:
@renatobecker Fixed this issue